markets/spot-market/contracts/modules/AsyncOrderModule.sol

```diff
@@ -115,21 +121,19 @@ contract AsyncOrderModule is IAsyncOrderModule {
         uint128 marketId,
         uint128 asyncOrderId
     ) external override returns (uint, int, uint) {
-        SpotMarketFactory.Data storage spotMarketFactory = SpotMarketFactory.load();
-        spotMarketFactory.isValidMarket(marketId);
```

Comment:
this check was pointless, we're loading the claim below based on market id and if it doesn't exist well it fails anyway